exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,17 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
```

Comment:
Great attempt Shima, you did this very well.

Unfortunately, you do have a couple issues here. You start by using `mark >= 80` which means if the mark is 80 or higher.

Then after this you use `mark > 60` and `mark < 80`. Unfortunately these conditions **exclude** 60 and 80 since they don't have the equals sign.

You also don't need to check if the mark is less than 80 on line 14 because the first condition already shares that if it was 80 or higher it'd `return "A"`. So if it was 80 or 85 or 1120, the function would already return.

You can simplify your second condition by using

```javascript
} else if (mark >= 60) {
  return "B";
} else if ....
```